src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -31,41 +30,43 @@
 //   ** CourseCard should display authors list.
 //   ** CourseCard should display created date in the correct format.
 
-import React from 'react';
-
-import { getCourseDuration, formatCreationDate } from '../../../../helpers';
+import React from "react";
 
-export const CourseCard = ({course, handleShowCourse, authorsList}) => {
+import { getCourseDuration, formatCreationDate } from "../../../../helpers";
 
-		// write your code here
+import deleteIcon from "../../../../assets/deleteButtonIcon.svg";
+import editIcon from "../../../../assets/editButtonIcon.svg";
 
-	return (
-		<div className={styles.cardContainer} data-testid='courseCard'>
-			<div className={styles.cardText}>
-				<h2>Title</h2>
-				<p>Description</p>
-			</div>
-			<div className={styles.cardDetails}>
-				<p>
-					<b>Authors: </b>
-					authors list
-				</p>
-				<p>
-					<b>Duration:</b>
-					<span>duration</span>
-				</p>
-				<p>
-					<b>Created: </b>
-					<span>date</span>
-				</p>
-				<div>
+import styles from "./styles.module.css";
 
-					// reuse Button component for 'Show course' button
-					// reuse Button component for 'Delete' button with data-testid="deleteCourse"
-					// reuse Button component for 'Update' button with data-testid="updateCourse"
+export const CourseCard = ({ course, handleShowCourse, authorsList }) => {
+  // write your code here
 
-				</div>
-			</div>
-		</div>
-	);
+  return (
+    <div className={styles.cardContainer} data-testid="courseCard">
+      <div className={styles.cardText}>
+        <h2>Title</h2>
+        <p>Description</p>
+      </div>
+      <div className={styles.cardDetails}>
+        <p>
+          <b>Authors: </b>
+          authors list
+        </p>
+        <p>
+          <b>Duration:</b>
+          <span>duration</span>
+        </p>
+        <p>
+          <b>Created: </b>
+          <span>date</span>
+        </p>
+        <div className={styles.buttonsContainer}>
+          // reuse Button component for 'Show course' button 
+					// reuse Button component with deleteButtonIcon from 'src/assets' for 'Delete' button with data-testid="deleteCourse" 
+					// reuse Link component with editButtonIcon from 'src/assets' for 'Update' button with data-testid="updateCourse"
```

Comment:
// reuse Link component with editButton icon from 'src/assets' for 'Update' button with data-testid="updateCourse"